pkg/state/state.go

```diff
@@ -41,6 +41,13 @@ CREATE UNIQUE INDEX IF NOT EXISTS only_first_migration_without_parent ON %[1]s.m
 -- History is linear
 CREATE UNIQUE INDEX IF NOT EXISTS history_is_linear ON %[1]s.migrations (schema, parent);
 
+-- Add a column to tell whether the row represents an auto-detected DDL capture or a pgroll migration
+ALTER TABLE %[1]s.migrations ADD COLUMN IF NOT EXISTS migration_type
+  VARCHAR(32)
```

Comment:
would it make sense to use a boolean? do you plan for other uses here?

Reply:
I don't forsee other migration types, but I'm not sure if switching to a boolean here and losing the possibility of adding more is worth it.